app/src/main/res/values/help.xml

```diff
@@ -63,7 +56,7 @@ Time Travel mode enables you to use Sky Map to see the sky at times
 other than the present. Select \'Time Travel\' from the main menu, and
 then select one of the Popular Dates or choose your own date and time
 using the \'Change the Date\' and \'Change the Time\' dialogs. Selections
-are limited to the years 1900-2100 due to limitations on accuracy,
+are limited to the years 1900-2100 due to limitations on accuracy
```

Comment:
Hmmm.... "limited to .... due to limitations on..." is a bit repetitive. Any suggestions on how to word that better?